test/database/sqlite/schema_utils.py

```diff
@@ -282,6 +283,12 @@ def _add_with_references(filled: T, service: BaseService):
         if isinstance(filled, AuxiliaryEquipment):
             service.add(filled.terminal)
 
+        if isinstance(filled, CurrentTransformer):
+            service.add(filled.asset_info)
+
+        if isinstance(filled, PotentialTransformer):
+            service.add(filled.asset_info)
+
```

Comment:
Not super familiar with how this works but do you also need CurrentTransformerInfo and PotentialTransformerInfo covered here?

Reply:
No, this function is only responsible for adding a filled object along with the objects it directly references to a service (e.g. CurrentTransformer along with its CurrentTransformerInfo). Because asset infos do not backreference their asset, CTI and VTI do not need cases here.
In the case of PowerTransformerInfo, TransformerTankInfo, and TransformerEndInfo, they have cases here because of direct references between those classes.

Reply:
ah okay makes sense.